packages/@lightningjs/ui-components/src/mixins/withUpdates/index.js

```diff
@@ -49,6 +49,20 @@ function getPropertyDescriptor(name, key) {
   };
 }
 
+function getAliasPropertyDescriptor(prev, curr) {
+  const deprecationWarning = `The property "${prev}" is deprecated. Please use "${curr}" instead.`;
```

Comment:
same here as above - should we add something like "and will be removed in a future version"? idk if that's implied or if being more explicit is better

Reply:
updated to match the above